CRM/ManualDirectDebit/Mail/DataCollector/Base.php

```diff
@@ -203,6 +216,37 @@ private function collectRecurringContributionData() {
     ];
   }
 
+  /**
+   * Builds a HTML table for recurring contribution installments
```

Comment:
This class' purpose is really to only collect the data so that it can be used later when building the template. So here, `$recurringContributionRows['recurringInstallmentsTable'] = $recurringContributionPlan;` should be enough, and we build the HTML elsewhere. However, I understand this will not work in this case because of the problem you found with the wysiwyg editor, but this will definitely not be obvious for anyone else. So it might be a good idea to document this problem in this method's description, so other devs can understand why this had to be done.